third_party/tcplp/bsdtcp/tcp_subr.c

```diff
@@ -352,7 +365,7 @@ struct tcpcb *
 tcp_drop(struct tcpcb *tp, int errnum)
 {
 	if (TCPS_HAVERCVDSYN(tp->t_state)) {
-		tcp_state_change(tp, TCPS_CLOSED);
+		tcp_state_change(tp, TCP6S_CLOSED);
```

Comment:
I aliased the state constants in the FreeBSD implementation, so `TCP6S_TIME_WAIT` is the same as `TCPS_TIME_WAIT`, and `TCP6S_CLOSED` is the same as `TCPS_CLOSED`. Those changes you have seem correct, but aren't strictly necessary. Happy to standardize on using one set of constants or the other.

Reply:
Since your previous code mixed `TCP6S_` and `TCPS_`, when I first read `TCPS_`, I misunderstood that it was for IPv4. So I think it is necessary to use `TCP6S_` uniformly.